src/main/java/hudson/plugins/sshslaves/SSHConnector.java

```diff
@@ -279,7 +279,8 @@ public String getDisplayName() {
         }
 
         public ListBoxModel doFillCredentialsIdItems(@AncestorInPath ItemGroup context) {
-            if (!(context instanceof AccessControlled ? (AccessControlled) context : Jenkins.getInstance()).hasPermission(Computer.CONFIGURE)) {
+            AccessControlled _context = (context instanceof AccessControlled ? (AccessControlled) context : Jenkins.getInstance());
+            if (_context != null && !_context.hasPermission(Computer.CONFIGURE)) {
```

Comment:
I am not sure how this would _add_ a security hole, but it's true that if `Jenkins.getInstance()` returns null, the behaviour should be to just return the empty `ListBoxModel`, so I have changed it anyway.